docs/source/index.rst

```diff
@@ -15,6 +15,7 @@ Guide
    :maxdepth: 3
 
    readme
+   /tutorial/tutorial
```

Comment:
What is this referencing? I don't see any tutorial file but I don't know sphinx that well so I might be missing something.

Reply:
yes, sorry, this is taken care of in the tutorial branch, which is also currently being worked on.